packages/cdk/lib/stacks/context-stack.ts

```diff
@@ -25,18 +26,33 @@ export class ContextStack extends Stack {
 
     const { contextParameters } = props;
     const { engineName } = contextParameters;
+    const { filesystemType } = contextParameters;
+    const { fsProvisionedThroughput } = contextParameters;
+    this.iops = Size.mebibytes(fsProvisionedThroughput!);
 
     switch (engineName) {
       case ENGINE_CROMWELL:
+        if (filesystemType != "S3") {
```

Comment:
I'm a bit confused here. What's the point of exposing `filesystemType` if we error on any non-default values?

Reply:
I think the plan here is to have other filesystem types in the future.